store/watcher_hub.go

```diff
@@ -31,9 +31,9 @@ import (
 // event happens between the end of the first watch command and the start
 // of the second command.
 type watcherHub struct {
+	count        int64 // current number of watchers.
 	mutex        sync.Mutex
```

Comment:
count is protected by the mutex right? move the mutex before the count? or at least add a comment